scss/texts.scss

```diff
@@ -1,11 +1,10 @@
-$gray : #828282;
-
-.text-1,
-.text-1-description {
+.text-1 {
   font-size: 20px;
   line-height: 24px;
 }
 
-.text-1-description {
+.text-2 {
+  font-size: 20px;
```

Comment:
When as possible, the font sizes should be used from variables and declared in another file.